{"metadata":{"timestamp":"2025-02-27T10:12:13Z"},"data":[{
      "cve": {
        "id": "CVE-2022-36017",
        "description": "TensorFlow is an open source platform for machine learning. If `Requantize` is given `input_min`, `input_max`, `requested_output_min`, `requested_output_max` tensors of a nonzero rank, it results in a segfault that can be used to trigger a denial of service attack. We have patched the issue in GitHub commit 785d67a78a1d533759fcd2f5e8d6ef778de849e0. The fix will be included in TensorFlow 2.10.0. We will also cherrypick this commit on TensorFlow 2.9.1, TensorFlow 2.8.1, and TensorFlow 2.7.2, as these are also affected and still in supported range. There are no known workarounds for this issue.",
        "metadata": {
          "published_date": "2022-09-16T23:15:11.367",
          "last_modified_date": "2024-11-21T07:12:11.560",
          "confidence_level": "low",
          "severity": "high"
        },
        "impact": {
          "cisa_kev": false,
          "reported_exploited": false,
          "exploit_maturity": "none",
          "automatable": true
        },
        "counts": {
          "public_exploit_count": 0
        },
        "epss": {
          "score": 0.00075,
          "percentile": 0.35337
        },
        "metrics": [
          {
            "source": "nvd@nist.gov",
            "type": "Primary",
            "cvss_version": "3.1",
            "vector_string": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
            "attack_vector": "NETWORK",
            "base_score": 7.5
          },
          {
            "source": "security-advisories@github.com",
            "type": "Secondary",
            "cvss_version": "3.1",
            "vector_string": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H",
            "attack_vector": "NETWORK",
            "base_score": 5.90000009536743
          }
        ],
        "timeline": {}
      }
    }]}